frontend/docs/css-style.guide.md

```diff
@@ -0,0 +1,277 @@
+
+There are particular ocassions where using ids is the only choice (i.e.: You want to override some rules from a component).
+
+If a classname should have 2 words, they must be separed only by a middle dash (hyphen-case a.k.a. kebab-case)
+
+**.demo-image** instead of **.demo_image**, nor **.demoImage** nor **.demoimage**
+
+#### CSS and HTML tags:
+Avoid using HTML tags en css:
+Instead of **p.error{ }** , use **.error{ }**
+Instead of **p {}**, include in your html **p.caption** and use **.caption{}**
+
+#### Shorthands:
+Shorthands use is expected
+
+    margin: 0 10px 20px;
+
+Shorthands (used for margin, padding, border, etc.) depends on the amount of values listed.
+
+If a shorthand has 4 values
+
+    margin: 25px 50px 75px 100px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 75px;
+    margin-left: 100px;
+
+If a shorthand has 3 values
+
+    margin: 25px 50px 75px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 75px;
+    margin-left: 50px;
+
+If a shorthand has 2 values
+
+    margin: 25px 50px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 25px;
+    margin-left: 50px;
+
+If a shorthand has 1 value
+
+    margin: 25px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 25px;
+    margin-bottom: 25px;
+    margin-left: 25px;
+
+
+#### Units:
+Avoid adding unit suffix when the value is **“0”**.
+
+    margin: 0;
+
+Instead of
+
+    margin: 0px;
+
+#### Fonts:
+Web safe fonts (A.K.A. callback fonts) are recommended
+
+    .caption {
+        font-family: 'Times New Roman', Times, serif;
+    }
+
+If the browser couldn't load the first font, it would use the next one and so on.
+
+#### Colors:
+Color variables must be used
+
+    color: #4285F4; //wrong
+    color: $cornflower-blue; //right
+
+Then, in a file called **\_colors.scss**
+
+    $cornflower-blue: #4285F4;
+
+Color name should describe the color as it is and not where is used.
+
+Avoid:
+
+    background-color: $table-header-color;
+
+Avoid:
+
+    color: $main-title;
+
+Instead, use:
+
+    color: $cornflower-blue;
+
+To avoid color repetition (Ej: _$light-grey_, _$lighter-grey_) we use [Name that color](http://chir.ag/projects/name-that-color/).
+
+Use color abbreviations if possible, changing
+
+    $beauty-bush: #EEBBCC;
+
+for
+
+    $beauty-bush: #EBC;
+
+Color hex should not be in lowercase, changing
+
+    $beauty-bush: #ebc;
+
+for
+
+    $beauty-bush: #EBC;
+
+#### Properties order:
+Properties must be alphabetically ordered
+
+    background: fuchsia;
+    border: 1px solid;
+    border-radius: 4px;
+    color: black;
+    text-align: center;
+    text-indent: 2em;
+
+#### Indentation:
+Property should be indented with 2 spaces respect from its containing block
+
+    .test {
+      display: block;
+      height: 100px;
+    }
+
+In case of nested blocks, they should be separed with one blank link  and indented with 2 spaces
+
+    .test {
+
```

Comment:
This is just a suggestion, but I would add a property to .test so that the point is clearer (if you read it quickly, you may think this apply to simple classes, like:

.test {

  display:block;
  height: 100px;
}

which is not what we want)